README.md

```diff
@@ -83,3 +90,12 @@ Type `"` in any `pipeline.py` and it should trigger the autocompletion list.
 ## Hover
 Just hover your mouse over any `params:`, datasets or hit the command `Show or Focus Hover`
 ![hover](assets/lsp-hover.gif)
+
+## Flowchart to Editor navigation
+
+Clicking on a node in flowchart navigate to the corresponding node function in the code.
+![navigation to node function](assets/viz-vsc-nav-function-node.gif)
+
+
+Clicking on a data node in flowchart navigate to the corresponding dataset yaml file.
+![navigation to dataset](assets/viz-vsc-nav-data-node.gif)
```

Comment:
To navigate between the flowchart and the code editor in Kedro-Viz:

1. Navigate to Node Functions:

Click on a node in the Kedro-Viz flowchart, and it will automatically navigate to the corresponding node function in your code.

2. Navigate to DataCatalog:

Clicking on a data node in the flowchart will open the corresponding dataset in the Data Catalog.